packages/bpk-component-accordion/src/BpkAccordion.js

```diff
@@ -29,26 +29,36 @@ const getClassName = cssModules(STYLES);
 
 type Props = { children: Node, className: ?string };
 
+export const BpkAccordionContext = createContext({ onDark: false });
+
 const BpkAccordion = (props: Props) => {
   const { children, className, ...rest } = props;
 
-  const classNames = getClassName('bpk-accordion', className);
+  const classNames = getClassName(
+    'bpk-accordion',
+    props.onDark ? 'bpk-accordion--day' : undefined,
```

Comment:
Here you can just do 
```suggestion
    props.onDark && 'bpk-accordion--day',
```